apps/aecore/test/aecore_sync_SUITE.erl

```diff
@@ -310,6 +312,54 @@ check_no_system_metrics_sent() ->
                             nomatch =/= re:run(L, "^ae\\.epoch\\.system", [])]
       end, aec_metrics_test_utils:fetch_data()).
 
+%% It takes about 320 ms to fetch the blocks from the other chain.
+%% In that interval we need to have the sync process crash
+crash_syncing_worker(Config) ->
+    [ Dev1, Dev2 | _ ] = proplists:get_value(devs, Config),
+    N1 = aecore_suite_utils:node_name(Dev1),
+    N2 = aecore_suite_utils:node_name(Dev2),
+    {ok, DbCfg} = node_db_cfg(Dev2),
+    aecore_suite_utils:delete_node_db_if_persisted(DbCfg),
+    aecore_suite_utils:stop_node(Dev2, Config),
+    
+    Top1 = rpc:call(N1, aec_chain, top_block, [], 5000),
+    ct:log("top at Dev1 = ~p", [Top1]),
+    ExtraBlocks = 5,  %% Might need more if CPU is really fast!
+    aecore_suite_utils:mine_blocks(N1, ExtraBlocks),
+    H1 = aec_blocks:height(Top1) + ExtraBlocks,
+
+    %% Set the same mining_rate to validate target
+    application:set_env(aecore, expected_mine_rate, 10), 
+
+    aecore_suite_utils:start_node(Dev2, Config),
+    aecore_suite_utils:connect(N2),
+    ct:log("node connected ~p", [N2]),
+    ok = rpc:call(N2, application, set_env, [aecore, ping_interval_limits, {500, 1000}], 5000),
+
+    spawn_link(fun() -> kill_sync_worker(N2, <<"http://ThomasComputer:3013">>) end),
```

Comment:
This looks interesting - either the node name doesn't matter or else something is fishy?

Reply:
It doesn't really matter - it is going to be killed right away

Reply:
It matters... only when we kill the process that is syncing, we test the right thing. Updated this.